source/read.txt

```diff
@@ -13,5 +27,116 @@ Read Data from MongoDB
    /read/specify-documents-to-return
    /read/project
    /read/distinct
-   /read/project
    /read/count
+   .. TODO: /read/change-streams
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+{+driver-short+} methods for retrieving documents.
+
+.. tip::
+
+   To learn more about any of the methods shown on this page, see the link
+   provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <cpp-read-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as
+``<connection string>``, with the relevant values for your MongoDB
+deployment.
```

Comment:
S: Since there are placeholders in the code examples and in the sample app:

```suggestion
Be sure to replace all placeholders, such as
``<connection string>``, with the relevant values for your MongoDB
deployment.
```